src/coord/src/timestamp.rs

```diff
@@ -40,6 +40,8 @@ use ore::collections::CollectionExt;
 
 use crate::coord;
 
+const CONF_DENYLIST: &'static [&'static str] = &["statistics.interval.ms"];
```

Comment:
answered offline but for posterity -- i continued using this convention in the timestamper where it was already present but didn't feel like it needed to be ported over to sinks because we are only denying the one config option. going to merge as is, and can change to use a denylist if others feel strongly (Aljoscha said he didn't care either way)